Detectors/TPC/calibration/src/CalibratordEdx.cxx

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "TPCCalibration/CalibratordEdx.h"
+
+#include <array>
+#include <cstddef>
+#include <memory>
+#include <string_view>
+
+//o2 includes
+#include "CommonUtils/MemFileHelper.h"
+#include "CommonUtils/TreeStreamRedirector.h"
+#include "CCDB/CcdbApi.h"
+#include "DetectorsCalibration/Utils.h"
+#include "Framework/Logger.h"
+#include "TPCCalibration/CalibdEdxHistos.h"
+#include "TPCCalibration/CalibdEdx.h"
+
+using namespace o2::tpc;
+
+void CalibratordEdx::initOutput()
+{
+  // Here we initialize the vector of our output objects
+  mInfoVector.clear();
+  mMIPVector.clear();
+  return;
+}
+
+void CalibratordEdx::finalizeSlot(Slot& slot)
+{
+  LOG(INFO) << "Finalizing slot " << slot.getTFStart() << " <= TF <= " << slot.getTFEnd();
```

Comment:
Please change all logging message to `fmt::format` style using `LOGP(info|error|warning, "", ...)`
```suggestion
  LOGP(info, "Finalizing slot {} <= TF <= {}" , slot.getTFStart(), slot.getTFEnd());
```